2/2.46.2/Dockerfile

```diff
@@ -38,10 +38,10 @@ RUN mkdir -p /usr/share/doc/tini && curl -fsSL https://raw.githubusercontent.com
 COPY init.groovy /usr/share/jenkins/ref/init.groovy.d/tcp-slave-agent-port.groovy
 
 # jenkins version being bundled in this docker image
-ENV JENKINS_VERSION 2.32.2
+ENV JENKINS_VERSION 2.46.2
```

Comment:
hmm, should we just use `2/` directory? I don't think renaming the directory for every update is very covenient